validate.py

```diff
@@ -69,6 +69,12 @@ def get_reviews():
 
       print('  %s: %s at %s' % (user, state, commit))
       if state == 'APPROVED' and commit != target_commit:
+        # An approval clears out any past rejections from a user.
```

Comment:
Isn't this backwards? It looks to me like if there's an approval on a very early commit, it would potentially clear out a more recent rejection. (I don't know what order the reviews show up in `response.json`)

Reply:
Reviews in response.json come in chronological order

Reply:
Specifically, if we see a review while iterating through reviews, we know it's more recent than any  review we already saw